src/Processor/DateProcessor.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the BibTex Parser.
+ *
+ * (c) Renan de Lima Barbosa <renandelima@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace RenanBr\BibTexParser\Processor;
+
+
+use DateTime;
+
+class DateProcessor
+{
+    use TagCoverageTrait;
```

Comment:
Replace this trait by `TagSearchTrait`, then we can call `tagSearch()` to get the real tag name for `year` and `month`